src/ui.js

```diff
@@ -152,4 +191,68 @@ export default class UI {
   static taskCompleted(index, value) {
     taskCompleted(index, value);
   }
+
+  static changeLiToEditMode(li) {
+    let classesLi = li.className;
+    classesLi = classesLi.replace('appItem', 'appItemEdit');
+    li.className = classesLi;
+    const childrenLi = li.children;
+
+    // change clases of divs
+    const normalView = childrenLi[0];
+    let classesNV = normalView.className;
+    classesNV = classesNV.replace('d-flex', 'd-none');
+    normalView.className = classesNV;
+
+    const editView = childrenLi[1];
+    let classesE = editView.className;
+    classesE = classesE.replace('d-none', 'd-flex');
+    editView.className = classesE;
+
+    // show appropriate icons in edit view
+    const divIcons = childrenLi[2];
+    const icons = divIcons.children;
+    icons[0].classList.add('d-none');
+    this.changeClassToElement(icons[1], 'd-none', '');
+    this.changeClassToElement(icons[2], 'd-none', '');
+
+    // sets focus con the input to edit
+    const inputEdit = editView.querySelector('input');
+    inputEdit.id = 'inputEdit';
+    inputEdit.focus();
+  }
+
+  static changeClassToElement(item, oldClass, newClass) {
+    let classesItem = item.className;
+    classesItem = classesItem.replace(oldClass, newClass);
+    item.className = classesItem;
+  }
+
+  static removeTask(index) {
+    // gets todos from local storage
+    const todos = Store.getTasks();
+
+    todos.splice(index, 1);
+
+    Store.setTasks(todos);
+  }
+
+  static updateTask(index, newDesc) {
+    // gets todos from local storage
+    const todos = Store.getTasks();
+    // sets new description in respective index
+    todos[index].description = newDesc;
+    // sets new todos to storage
+    Store.setTasks(todos);
+  }
+
+  static clearCompleted() {
+    // gets todos from local storage
+    const tasks = Store.getTasks();
+
+    // create new array from filter()
+    const uncompletedTasks = tasks.filter(returnsUncompleted);
+
+    Store.setTasks(uncompletedTasks);
+  }
```

Comment:
- [ ] I would recommend making a separate `js` file for new functionality and import as a module.

Reply:
making a new js file for 3 lines of code? ok.....